handler_test.go

```diff
@@ -1175,14 +1182,39 @@ func TestHandler_UnmatchColumnID(t *testing.T) {
 				}
 			]
 		}`)
-	h := NewHandler()
-	h.Holder = hldr.Holder
-	h.Cluster = NewCluster(1)
-	w := httptest.NewRecorder()
-	h.ServeHTTP(w, MustNewHTTPRequest("POST", "/index/i0/input-definition/input1", bytes.NewBuffer(inputBody)))
+
+	w = httptest.NewRecorder()
+	h.ServeHTTP(w, MustNewHTTPRequest("POST", "/index/i1/input-definition/input1", bytes.NewBuffer(unmatchColumnBody)))
+	if w.Code != http.StatusBadRequest {
+		t.Fatalf("unexpected status code: %d", w.Code)
+	} else if body := w.Body.String(); body != pilosa.ErrInputDefinitionColumnLabel.Error()+"\n" {
+		t.Fatalf("unexpected body: %s", body)
+	}
+
+	// Eusure throwing error if request body is invalid
```

Comment:
Add a comment about what you are checking for in the invalid body.
I can see it from the JSON, but a specific comment may help future users since the error message is only 
"unexpected EOF"